src/main/java/kostka/moviecatalog/service/RedisService.java

```diff
@@ -1,16 +1,22 @@
 package kostka.moviecatalog.service;
 
+import kostka.moviecatalog.entity.Movie;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.redis.core.RedisTemplate;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Service
 public class RedisService {
-    static final String LATEST_MOVIES_KEY = "latestMovies";
+    public static final String LATEST_MOVIES_KEY = "latestMovies";
+    public static final String TOP_RATING_MOVIES_KEY = "topRatingMovies";
+    public static final long START = 0L;
+    public static final long LONG_4 = 4L;
```

Comment:
Jen bych doporučil jiné pojmenování konstant. Znázvu LONG_4 je stejné jako magické 4L. Není poznat k čemu to teda slouží.

Reply:
zkusil jsem LIMIT, snad to projde na review :)